Shared/Samples/Find nearest vertex/FindNearestVertexView.swift

```diff
@@ -0,0 +1,184 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct FindNearestVertexView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// A location callout placement.
+    @State var calloutPlacement: CalloutPlacement?
+    
+    /// The tap location.
+    @State var tapLocation: Point!
+    
+    var body: some View {
+        MapView(map: model.map, graphicsOverlays: [model.graphicsOverlay])
+            .onSingleTapGesture { _, mapPoint in
+                // Normalize map point.
+                guard let normalizedMapPoint = GeometryEngine.normalizeCentralMeridian(of: mapPoint) as? Point else { return }
+                tapLocation = normalizedMapPoint
+                if calloutPlacement == nil {
+                    // Draw the point graphics.
+                    model.drawNearestPoints(point: tapLocation)
+                    // Show the callout at the tapped location.
+                    calloutPlacement = CalloutPlacement.location(normalizedMapPoint)
```

Comment:
```suggestion
                    calloutPlacement = CalloutPlacement.location(tapLocation)
```
No difference in functionality, but reads more naturally.